src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectJobProperty/help-propertiesContent.html

```diff
@@ -1,7 +1,8 @@
 <div>
-    <p>
-        Give a set of key/value (KEY=VALUE).<br/>
-        All the properties name will be accessible as environment variables by their names.<br/>
-        You can use or override the properties specified in the above properties file.<br/>
-    </p>
+    <p>Give a set of key/value (one variable per line): <code>KEY=VALUE</code>.</p>
+
+    <p>All the properties name will be accessible as environment variables by their names. You can use or override the
+        properties specified in the above properties file.</p>
+
+    <p>These environment variables (<strong>only these</strong>) will be available in the context Job.</p>
```

Comment:
What is a <code>context Job</code>? I don't know such term. In any case it makes sense to add several sentences, which would allow users to understand what's going on here.